test/tests/p5.Amplitude.js

```diff
@@ -64,4 +63,3 @@ define(['chai'], function (chai) {
       }, 10);
     });
   });
-});
```

Comment:
keeping the indentation as-is is very helpful to review, but I think we'll want to format it before we merge.

We can do this with [prettier](https://prettier.io/)

`prettier test/*/js test/tests/*.js --write`

(this could probably be added to our lint task)

Reply:
i think we can have a separate lint task to collectively for all ?